OpenRA.Launcher/Program.cs

```diff
@@ -31,6 +31,7 @@ static int Main(string[] args)
 			}
 			catch (Exception e)
 			{
+				Log.Dispose();
```

Comment:
This now hangs when *not* crashing :grin:

Put it in a `finally`?

Reply:
doh!